lib/std/fs.zig

```diff
@@ -1757,7 +1817,17 @@ pub const Dir = struct {
         return Dir{ .fd = fd };
     }
 
-    fn openDirAccessMaskW(self: Dir, sub_path_w: [*:0]const u16, access_mask: u32, no_follow: bool) OpenError!Dir {
+    const OpenDirWithAccessAndCreationWOptions = struct {
+        no_follow: bool,
+        createDisposition: u32
```

Comment:
```suggestion
        create_disposition: u32
```

https://ziglang.org/documentation/master/#Names

Reply:
@squeek502 Thank you for the feedback, I have updated the PR to match your suggestion.